themes/edx.org/lms/templates/header.html

```diff
@@ -87,18 +87,28 @@ <h1 class="hd logo-header">
                     <a class="nav-link" href="${marketing_link('SCHOOLS')}">${_("Schools & Partners")}</a>
                   </li>
                 % endif
-                % if show_program_listing:
+                %if show_dashboard_tabs:
```

Comment:
We've got the same code 4 times?  No way to make this a reusable block with an argument to control the extra classes? If it doesn't get pulled out, you need to make the same fixes here as everywhere else.